code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -92,12 +92,36 @@
 	probability = 0
 	max_engineers = 8
 
+/datum/emergency_call/cryo_squad/tech // Actual techweb cryorines
+	name = "Marine Cryo Reinforcements (Tech)"
+	mob_max = 6
+	max_engineers = 2
+	max_medics = 2
+	max_smartgunners = 1
+	max_heavies = 0
+
 /obj/effect/landmark/ert_spawns/distress_cryo
 	name = "Distress_Cryo"
 
-/datum/emergency_call/cryo_squad/tech
-	name = "Marine Cryo Reinforcements (Tech)"
-	mob_max = 5
-	max_engineers = 1
-	max_medics = 1
-	max_heavies = 0
+/datum/emergency_call/cryo_squad/tech/proc/get_marines()
+	var/pop_scale = 0
+	var/count = 0
+
+	for (var/mob/living/carbon/human/H in GLOB.human_mob_list)
```

Comment:
if you want it to scale with server pop, use GLOB.clients, dont use for loop, just single var with length(GLOB.clients)